src/mlpack/methods/pca/pca_impl.hpp

```diff
@@ -74,6 +74,21 @@ void PCA<DecompositionPolicy>::Apply(const arma::mat& data,
   arma::mat eigvec;
   Apply(data, transformedData, eigVal, eigvec);
 }
+    
+/**
+ * Apply Principal Component Analysis to the provided data set.
+ *
+ * @param data - Data matrix
+ * @param transformedData - Data with PCA applied
```

Comment:
```suggestion
 * @param data - Data matrix.
 * @param transformedData Data with PCA applied.
```

Just to be consistent with the rest of the codebase.